GoogleForCreators/web-stories-wp
# Block: Use `getUsers` with capability query

## Context



## Summary

Replace `getAuthors` with `getUsers`. 

Followed this example - https://github.com/WordPress/gutenberg/pull/33695

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Create a post. 
2. Insert a web story block. 
3. Select "selected stories". 
4. Select a template. 
5. Click select stories. 
6. Click author dropdown. See that dropdown still works.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #9240
